pom.xml

```diff
@@ -114,6 +114,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.version>3.3.9</maven.version>
+    <maven.surefire.plugin.version>2.22.0</maven.surefire.plugin.version>
```

Comment:
The parent POM here already declares the surefire plugin, but version 2.20.1. That would need to be changed. If you do, you don't need a version in the child POM, nor do you really need a property to control the version and the extra indirection.

Reply:
Yes but I thought you wanted me to add that version at the parent pom. Otherwise I can simply remove the version from the child pom.